app/views/layouts/application.html.erb

```diff
@@ -79,22 +79,22 @@
         <div class="col-md-3">
           <ul>
             <li>
-              <a href="http://webchat.freenode.net/?channels=diaspora"><%= t '.irc_general' %></a>
+              <a href="https://webchat.freenode.net/?channels=diaspora"><%= t '.irc_general' %></a>
             </li>
             <li>
-              <a href="http://webchat.freenode.net/?channels=diaspora-dev"><%= t '.irc_development' %></a>
+              <a href="https://webchat.freenode.net/?channels=diaspora-dev"><%= t '.irc_development' %></a>
             </li>
           </ul>
         </div>
         <div class="col-md-3">
           <ul>
             <li>
-              <a href="http://groups.google.com/group/diaspora-discuss">
+              <a href="https://discourse.diasporafoundation.org/">
                 <%= t '.discussion_general' %>
               </a>
             </li>
             <li>
-              <a href="http://groups.google.com/group/diaspora-dev">
+              <a href="https://discourse.diasporafoundation.org/c/development">
```

Comment:
Do we need a separate link to the development category? If yes, maybe we should also add a link to the [support category](https://discourse.diasporafoundation.org/c/support). Otherwise we can just change the text for the "general" link to "[Discussion and Support](https://discourse.diasporafoundation.org/)"?